conda/install.py

```diff
@@ -56,7 +56,9 @@ def __enter__(self):
         def __exit__(self, exc_type, exc_value, traceback):
             pass
 
-on_win = bool(sys.platform == 'win32')
+from conda.utils import win_path_to_unix
```

Comment:
Thanks for pointing this out.  I have taken a similar approach that you did: try to import it, but fall back to defining it.